modules/ROOT/pages/download/nb25/index.adoc

```diff
@@ -0,0 +1,151 @@
+link:{url-download-keychecksum}netbeans-installers/{netbeans-version}/Apache-NetBeans-{netbeans-version}.pkg.asc[PGP ASC])
+* link:{url-download}netbeans-installers/{netbeans-version}/apache-netbeans_{netbeans-version}-1_all.deb[apache-netbeans_{netbeans-version}-1_all.deb] (link:{url-download-keychecksum}netbeans-installers/{netbeans-version}/apache-netbeans_{netbeans-version}-1_all.deb.sha512[SHA-512],
+link:{url-download-keychecksum}netbeans-installers/{netbeans-version}/apache-netbeans_{netbeans-version}-1_all.deb.asc[PGP ASC])
+* link:{url-download}netbeans-installers/{netbeans-version}/apache-netbeans-{netbeans-version}-0.noarch.rpm[apache-netbeans-{netbeans-version}-0.noarch.rpm] (link:{url-download-keychecksum}netbeans-installers/{netbeans-version}/apache-netbeans-{netbeans-version}-0.noarch.rpm.sha512[SHA-512],
+link:{url-download-keychecksum}netbeans-installers/{netbeans-version}/apache-netbeans-{netbeans-version}-0.noarch.rpm.asc[PGP ASC])
+* link:https://snapcraft.io/netbeans[Linux snap package]
+
+*Source:*
+
+* link:{url-download}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip[netbeans-{netbeans-version}-source.zip] 
+(link:{url-download-keychecksum}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip.sha512[SHA-512], 
+link:{url-download-keychecksum}netbeans/{netbeans-version}/netbeans-{netbeans-version}-source.zip.asc[PGP ASC])
+
+Officially, it is important that you link:https://www.apache.org/dyn/closer.lua#verify[verify the integrity]
+of the downloaded files using the PGP signatures (.asc file) or a hash (.sha512 files).
+The PGP keys used to sign this release are available link:https://downloads.apache.org/netbeans/KEYS[here].
+
+*Release Notes:*
+
+* link:https://github.com/apache/netbeans/releases/tag/{netbeans-version}[Github Link]
+
+== Community Installers
+
+- link:https://www.codelerity.com/netbeans/[Codelerity / Gj IT packages] - Windows, macOS and
+Linux (.deb / .rpm / .AppImage) built with
+link:https://github.com/apache/netbeans-nbpackage/[NBPackage]. Most
+include a local JDK runtime for the IDE to run on, for a self-contained out-of-the-box
+experience.
+
+IMPORTANT: Individual NetBeans committers may provide additional binary packages as a convenience.
+While built using the Apache NetBeans release, they are not releases of the Apache Software
+Foundation. They may include other contents (eg. JDK) under additional license terms.
+
+== Deployment Platforms
+
+The Apache NetBeans {netbeans-version} binary releases require JDK 17+, and officially support running on JDK 17, 21 and 24.
+
+TIP: The Runtime JDK NetBeans uses does not influence the JDK range projects can use.
+
+TIP: Use the latest JDK update release for the best experience. link:https://javaalmanac.io/jdk/[java almanac release index]
```

Comment:
this is new under deployment platforms.

I am not sure if we can link to the external page, but i can remove the link if there are concerns.